release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -158,9 +158,352 @@ For more information, see xref:../applications/creating_applications/creating-ap
 [id="ocp-4-14-ibm-z"]
 === {ibmzProductName} and {linuxoneProductName}
 
+With this release, {ibmzProductName} and {linuxoneProductName} are now compatible with {product-title} {product-version}. The installation can be performed with z/VM or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Installing a cluster with z/VM on {ibmzProductName} and {linuxoneProductName}]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc#installing-restricted-networks-ibm-z[Installing a cluster with z/VM on {ibmzProductName} and {linuxoneProductName} in a restricted network]
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-ibm-z-kvm[Installing a cluster with {op-system-base} KVM on {ibmzProductName} and {linuxoneProductName}]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc#installing-restricted-networks-ibm-z-kvm[Installing a cluster with RHEL KVM on {ibmzProductName} and {linuxoneProductName} in a restricted network]
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}
+====
+
+[discrete]
+==== {ibmzProductName} and {linuxoneProductName} notable enhancements
+
+The {ibmzProductName} and {linuxoneProductName} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibmzProductName} and {linuxoneProductName}:
+
+* Assisted Installer with z/VM
+* Installing on a single node
+* Hosted control planes (Technology Preview)
+* Multi-architecture compute nodes
+* oc-mirror plugin
+
+[discrete]
```

Comment:
@holgwolf Shall we keep this entry from the 4.13 RNs or is it sufficient to have SE in the supported features table.